### PR TITLE
pvanet for classfication on Imagenet

### PR DESCRIPTION
**Pull request by qingzhouzhen**
article adress : [Pvanet:Deep but Lightweight Neural Neural Networks for Real-time Object Detection](https://arxiv.org/abs/1608.08021)
pvanet for classification on ImageNet, the accuracy of the article is 70%, I checked the shape of parameter to make sure the net is same with the author implement with Caffe, but I still got a accuracy of 66.28%, I trained with:
  intialize：mx.init.Normal()
  batch-size=512
  lr-factor=0.316
  lr=0.1
  num_epochs=160
  lr_step_epochs='30,60,80,100,120,140'

result is :

  INFO:root:Epoch[158] Batch [2350]       Speed: 624.06 samples/sec       accuracy=0.696992       cross-entropy=1.213310  top_k_accuracy_5=0.890508
  INFO:root:Epoch[158] Batch [2400]       Speed: 624.90 samples/sec       accuracy=0.698438       cross-entropy=1.205256  top_k_accuracy_5=0.894531
  INFO:root:Epoch[158] Batch [2450]       Speed: 625.89 samples/sec       accuracy=0.690547       cross-entropy=1.232207  top_k_accuracy_5=0.890508
  INFO:root:Epoch[158] Batch [2500]       Speed: 627.79 samples/sec       accuracy=0.695352       cross-entropy=1.215836  top_k_accuracy_5=0.892695
  INFO:root:Epoch[158] Train-accuracy=0.683594
  INFO:root:Epoch[158] Train-cross-entropy=1.284250
  INFO:root:Epoch[158] Train-top_k_accuracy_5=0.888672
  INFO:root:Epoch[158] Time cost=2048.858
  INFO:root:Saved checkpoint to "pvanet-models/pvanet-0159.params"
  INFO:root:Epoch[158] Validation-accuracy=0.662847
  INFO:root:Epoch[158] Validation-cross-entropy=1.374140 
  INFO:root:Epoch[158] Validation-top_k_accuracy_5=0.869958

can this be accepted by MXNet?

### CODE REVIEW

**Comment by qingzhouzhen** [comment]
And if I want to contribute classification model, where should I put the code? just put the symbol in [https://github.com/apache/incubator-mxnet/tree/master/example/image-classification/symbols](https://github.com/apache/incubator-mxnet/tree/master/example/image-classification/symbols)? beside the symbol code, I will change some code such as optimizer, should I push these code?

**Comment by zhreshold** [comment]
Symbol is fine in the folder you mentioned. If you have optimizer to push, put them in python/mxnet/optimizer.py if you think these optimizers are widely useful to other users.

**Comment by zhreshold** [comment]
Can you figure out what's the difference with the one in caffe? Like data augmentation difference that caused the degraded perf?
